### PR TITLE
SOE-1413: updated font stylesheet

### PR DESCRIPTION
**Pull request by cjwest**
This updates the stylesheets importing the Roboto Slab and other fixes.

### CODE REVIEW

File: modules/stanford_jse_dept/stanford_jse_dept.module [hunk 1]
```diff
@@ -78,4 +78,39 @@ function stanford_jse_dept_css_alter(&$css) {
       break;
     default:
   }
-}
\ No newline at end of file
+
+  // Exclude admin pages
```
Comment (sherakama) [inline modules/stanford_jse_dept/stanford_jse_dept.module]: From line 82 to line 114 it does not appear to be doing anything. Please remove or revise.

Comment (cjwest) [reply modules/stanford_jse_dept/stanford_jse_dept.module]: This code is to keep the css in css files included above from affecting the pages listed. 
It's quite striking and a little unnerving when the CSS is applied to these admin pages.

Comment (sherakama) [reply modules/stanford_jse_dept/stanford_jse_dept.module]: Then why is the part that saves the rule commented out?

Comment (cjwest) [reply modules/stanford_jse_dept/stanford_jse_dept.module]: I refactored it.

**Pull request merged**